warehouse/templates/pages/help.html

```diff
@@ -55,6 +56,7 @@ <h1 class="page-title">Common Questions</h1>
         <h2 class="faq-group__first"><a href="#basics">Basics</a></h2>
         <ul>
           <li><a href="#packages">{{ packages() }}</a></li>
+          <li><a href="#trove-classifier">{{ trove_classifier() }}</a></li>
```

Comment:
I'd put the trove classifier item below "How do I package and publish my code for PyPI?".